Objects/Objects/BuiltElements/Archicad/Property.cs

```diff
@@ -0,0 +1,162 @@
+using System.Collections.Generic;
+using Speckle.Core.Kits;
+using Speckle.Core.Models;
+
+namespace Objects.BuiltElements.Archicad;
+
+public static class PropertyHelper
+{
+  private static readonly HashSet<char> DisallowedPropNameChars = new() { '.', '/' };
+
+  public static string RemoveDisallowedPropNameChars(string propertyName)
+  {
+    foreach (char c in DisallowedPropNameChars)
+    {
+      propertyName = propertyName.Replace(c, ' ');
+    }
+
+    return propertyName;
+  }
```

Comment:
It would be interesting if we can coordinate with @clairekuang on how disallowed characters are dealt with in Rhino/Acad. And possibly align this?

This doesn't seem to be strictly "property related" but rather part of `Base` right? Would it make sense to discuss aligning this at the SDK level?

Reply:
As a first step I moved this function to Core.

Reply:
Ok i'll open a ticket for this and ping you guys there